src/utils/embedded/embedded.utils.ts

```diff
@@ -205,6 +205,50 @@ async function insecurelyValidateApplication() {
 // Validate immediately on load
 // insecurelyValidateApplication();
 
+const getAbsoluteUrl = (url: string): string | null => {
+  if (!url) {
+    return null;
+  }
+  try {
+    return new URL(url, window.location.origin).href;
+  } catch {
+    return null;
+  }
+};
+
+export function getAppLogo(): string | null {
+  try {
+    const faviconSelectors = [
+      "link[rel='icon'][type='image/svg+xml']",
+      "link[rel='icon'][sizes='32x32']",
```

Comment:
Maybe it's better to remove the selectors with `sizes` and instead change `document.querySelector` to `document.querySelectorAll`, then sort by the sizes attribute (when available).